scripts/components/GviHomeCategoryItem.ts

```diff
@@ -0,0 +1,23 @@
+import Image from '@smartface/native/ui/image';
+import GviHomeCategoryItemDesign from 'generated/my-components/GviHomeCategoryItem';
+
+export default class GviHomeCategoryItem extends GviHomeCategoryItemDesign {
+    pageName?: string | undefined;
+    constructor(props?: any, pageName?: string) {
+        // Initalizes super class for this scope
+        super(props);
+        this.pageName = pageName;
+    }
+    get categoryImage(): any {
+        return this.imgLeft.image;
```

Comment:
you dont have to return image instance just return the string assigned

Reply:
done.